From c42787d8f218a4f7032b55970e1beaa7e978a9a7 Mon Sep 17 00:00:00 2001
From: Rodrigo Gryzinski <rogryza@rogryza.com>
Date: Sun, 15 Nov 2020 18:00:00 -0300
Subject: [PATCH] WUT

---
 config.def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index caf9755..4eaf367 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "DejaVu Sans Mono:pixelsize=18:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
-- 
2.29.2

